src/App.js

```diff
@@ -15,36 +15,23 @@ const getPokemons = async () => {
   let data = await res.json()
   setNextPage(data.next) // display 20 more pokemon (btn in future)
   setPokeName(data.results.map(p => p.name))
-
-  function createPokemonObject(result) {
-    result.forEach(async (pokemon) => {
-      let res = await fetch(`https://pokeapi.co/api/v2/pokemon/${pokemon}`)
-      let data = await res.json()
-      setAllPokemon(currentList => [...currentList, data])
-    })
-  }
-
-  createPokemonObject(pokeName)
 }
 
 
 useEffect(() => {
   getPokemons()
 }, [])
 
-
-
-console.log(allPokemon) 
-console.log(pokeName) // display on first load
+console.log(pokeName)
```

Comment:
when logging something add some label like
```suggestion
console.log('pokeName in App.js file is:', pokeName)
```